classes/Component.js

```diff
@@ -8,7 +8,7 @@ const { capitalize } = require('../utils');
 class Component {
   constructor(component, options) {
     this.component = capitalize(component);
-    this.options = options;
+    this.options = options || {};
```

Comment:
If `options` param has value or is `undefined` (not null) you can use default value just in params line
```
constructor(component, options = {}) {
...
}
```